types/three/three-core.d.ts

```diff
@@ -1447,6 +1447,15 @@ export class Geometry extends EventDispatcher {
     dispatchEvent(event: { type: string; [attachment: string]: any; }): void;
 }
 
+/**
+ * @see <a href="https://github.com/mrdoob/three.js/blob/master/examples/js/BufferGeometryUtils.js">examples/js/BufferGeometryUtils.js</a>
+ */
+export namespace BufferGeometryUtils {
+    export function mergeBufferGeometries(geometries: any[]): BufferGeometry;
```

Comment:
Is this an array of `any`?

Reply:
That is an array of `any`, but it (probably) should be an array of `BufferGeometry` as stated here: https://threejs.org/docs/#examples/BufferGeometryUtils

```typescript
export function mergeBufferGeometries(geometries: BufferGeometry[]): BufferGeometry;
```

Reply:
Great! Could you make those changes?

Reply:
Done.